envbuilder.go

```diff
@@ -518,11 +523,16 @@ func Run(ctx context.Context, options Options) error {
 				RegistryMirrors: registryMirror,
 			},
 			SrcContext: buildParams.BuildContext,
-		})
+		}
+		image, err := executor.DoBuild(opts)
```

Comment:
I'd like to see more debugging/trace logging around this step and `DoPush`.

Reply:
I'm not sure what the ask is here, I didn't change any of this code so seems out of scope?

Reply:
Didn't you add `image, err := executor.DoBuild(opts)`?

Reply:
No, I just moved opts to a variable.

Reply:
I missed L483. Anyway, I wanted to improve logging around this, but if you think this is useless here, I'm cool with leaving it as is.